core/src/main/java/io/kestra/core/utils/Rethrow.java

```diff
@@ -117,4 +120,17 @@ public static <E extends Exception> Runnable throwRunnable(RunnableChecked<E> ru
     private static <E extends Exception, R> R throwException(Exception exception) throws E {
         throw (E) exception;
     }
+
+    public static <T> Consumer<T> failAwareConsumer(Consumer<T> consumer, boolean failSilently) {
```

Comment:
As it's a bad practice to silently fail exception, please don't add it here but instead directly inside the JdbExecutor